app/templates/find_suppliers_and_services.html

```diff
@@ -0,0 +1,115 @@
+{% extends "_base_page.html" %}
+{% block page_title %}
+  Digital Marketplace admin
+{% endblock %}
+
+{% if current_user.role == "admin" %}
+  {% set page_title = "Edit supplier accounts or view services" %}
+{% elif current_user.role == "admin-ccs-category" %}
+  {% set page_title = "Edit suppliers and services" %}
+{% elif current_user.role == "admin-ccs-sourcing" %}
+  {% set page_title = "Edit supplier declarations" %}
+{% elif current_user.role == "admin-framework-manager" %}
+  {% set page_title = "View suppliers and services" %}
+{% endif %}
+
+{% block breadcrumb %}
+  {%
+      with items = [
+          {
+              "link": url_for('.index'),
+              "label": page_title
+          },
+      ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+  {% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+      {% for category, message in messages %}
+        {% if message['no_service'] %}
+          {% set displayed_message = "Could not find a service with ID: {}".format(message['no_service']) %}
+        {% elif message['api_error'] %}
+          {% set displayed_message = "Error trying to retrieve service with ID: {}".format(message['api_error']) %}
+        {% endif %}
+        {%
+          with
+          message = displayed_message,
+          type = "destructive" if category == 'error' else "success"
+        %}
+          {% include "toolkit/notification-banner.html" %}
+        {% endwith %}
+      {% endfor %}
+    {% endif %}
+  {% endwith %}
+  {%
+    with heading = page_title
+  %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form action="{{ url_for('.find_suppliers') }}" method="get" class="question">
+        {%
+          with
+          question = "Find a supplier by name",
+          name = "supplier_name_prefix",
+          hint = "You don't have to put in the full name to get a result."
```

Comment:
Should there be a curly apostrophe for `don't`?

Reply:
:eyes: Yes - definitely.